frontend-react/package.json

```diff
@@ -82,7 +82,7 @@
         "eject": "react-scripts eject",
         "lint": "npm-run-all -p lint:eslint",
         "lint:write": "npm-run-all -p lint:eslint:write",
-        "lint:eslint": "eslint \"src/**/*.{js,ts,jsx,tsx}\"",
+        "lint:eslint": "eslint \"src/**/*.{js,ts,jsx,tsx}\" --ignore-pattern \"src/typings/api-codegen.d.ts\"",
```

Comment:
Do we need this _and_ the line down on `196`? And was there a specific reason we didn't want to lint the generated code?

Reply:
Yes, the lint:eslint command doesn't take the ignore rule into account on 196 (the IDE does though I think).

Reply:
Ahhh, interesting 🤔 I wonder why that is. Good catch!

Reply:
I think the issue is that the file that's generated is at `src/typings/api-codgen.ts`.  If we change line 196 to `"src/typings/api-codegen.ts"` (note the `src/` path as well), we shooooooould be able to get rid of this ignore 🤔